keps/sig-api-machinery/0006-apply.md

```diff
@@ -143,11 +148,6 @@ The linked documents should be read for a more complete picture.
 
 (TODO: update this section with current design)
 
-What are the caveats to the implementation?
```

Comment:
I suspect we could entirely remove that section, but probably in a different PR.